README.md

```diff
@@ -4,6 +4,36 @@
 
 This is a stress testing tool for [BigBlueButton](https://bigbluebutton.org/).
 
+It simulates client activity (listening only, with microphone and/or webcam) thanks to [Chromedriver](https://chromedriver.chromium.org/) and [Selenium Python bindings](https://selenium-python.readthedocs.io/).
+Each client is a browser running in a virtual framebuffer (with `xvfb`), launched inside a [Docker](http://docker.io/) container.
+
+The audio input is generated with [espeak](http://espeak.sourceforge.net/) and [Fortune-mod](https://launchpad.net/fortune-mod). The webcam input is generated by [Google Chrome](https://www.google.com/chrome/).
+
+
+## Getting Started
+
+### Preparation
+
+1) Clone this repository
+
+2) Run `make .env` and update the `.env` file to specify `BBB_URL` and `BBB_SECRET`.\
+You can get these values by running `bbb-conf --secret` on your BBB server.
```

Comment:
`make build` or `make bootstrap` is missing here no ?

Reply:
The `bootstrap` and `build` target are for development purpose only, so I did'n mentionned them yet in the README.md.

The Getting started section describe how to start a test with the docker image published on DockerHub.